core/gesture.js

```diff
@@ -651,11 +651,25 @@ Blockly.Gesture.prototype.handleWsStart = function(e, ws) {
   this.setStartWorkspace_(ws);
   this.mostRecentEvent_ = e;
   this.doStart(e);
+  if (!this.startBlock_ && !this.startBubble_) {
+    this.fireWorkspaceClick_(ws);
```

Comment:
Curious why we don't do this in `doWorkspaceClick_`?

Reply:
woops! Just didn't know it was there.